src/main/scala/com/microsoft/ml/spark/lightgbm/TrainParams.scala

```diff
@@ -60,8 +61,8 @@ abstract class TrainParams extends Serializable {
 case class ClassifierTrainParams(parallelism: String, topK: Int, numIterations: Int, learningRate: Double,
                                  numLeaves: Int, maxBin: Int, binSampleCount: Int,
                                  baggingFraction: Double, posBaggingFraction: Double, negBaggingFraction: Double,
-                                 baggingFreq: Int, baggingSeed: Int, earlyStoppingRound: Int, featureFraction: Double,
-                                 maxDepth: Int, minSumHessianInLeaf: Double,
+                                 baggingFreq: Int, baggingSeed: Int, earlyStoppingRound: Int, improvementTolerance: Double, 
```

Comment:
looks like there is whitespace at end of line here that needs to removed.  Also line is too long.  You can just add newline after the earlyStoppingRound parameter.

Reply:
Done, sorry I fixed merge issues directly on github, so I did not get the highlights of the issues